docs/integrations/mirroring.md

```diff
@@ -193,17 +194,17 @@ def get_mapping_fields_command():
 
 ## Incident fields on a XSOAR incident 
 The following incident fields must be configured either in the integration or the instance mapping:
-* dbotMirrorDirection - valid values are Both, In, or Out.
-* dbotMirrorId - represents the id of the incident in the external system.
-* dbotMirrorInstance - the instance through which you will be mirroring.
+* **dbotMirrorDirection** - valid values are Both, In, or Out.
+* **dbotMirrorId** - represents the id of the incident in the external system.
+* **dbotMirrorInstance** - the instance through which you will be mirroring.
 
 Useful fields:
-* dbotMirrorTags - tags for mirrored-out entries (comment/files).
+* **dbotMirrorTags** - tags for mirrored-out entries (comment/files).
```

Comment:
Also:
- `dbotDirtyFields` for fields we changed locally and hence will not be updated from remote
- `dbotCurrentDirtyFields` for fields that are going to be sent remotely as delta in the next iteration of the job - probably should remain invisible to the end user
- `dbotMirrorLastSync` timestamp that includes last time we synched this incident with remote system